scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -57,7 +57,8 @@
             [metrics.histograms :as histograms]
             [metrics.meters :as meters]
             [metrics.timers :as timers]
-            [plumbing.core :as pc])
+            [plumbing.core :as pc]
```

Comment:
We try to keep imports in lexiconographic order.

Reply:
Ah, this was an ide auto-import. Will fix!